views/web.py

```diff
@@ -88,6 +92,34 @@ def me_or_explore(user):
         return redirect(url_for('explore', user=username))
 
 
+@app.route('/l/<string:link_id>', methods=['GET'])
+@user_required
+def love_link(link_id):
+    try:
+        loveLink = logic.love_link.get_love_link(link_id)
+        recipients_str = loveLink.recipient_list
+        message = loveLink.message
+
+        recipients = sanitize_recipients(recipients_str)
+        loved = [
+            Employee.get_key_for_username(recipient).get()
+            for recipient in recipients
+        ]
+
+        return render_template(
+            'love_link.html',
+            current_time=datetime.utcnow(),
+            current_user=Employee.get_current_employee(),
+            recipients=recipients_str,
+            message=message,
+            loved=loved,
+            link_id=link_id,
+        )
+    except NoSuchLoveLink:
+        flash('Sorry, that link ({}) is no longer valid.'.format(link_id), 'error')
+        return redirect(url_for('home'))
+
+
 @app.route('/explore', methods=['GET'])
 @user_required
 def explore():
```

Comment:
@rockdog yeah, for sure. alinar. would you mind pulling these? I think you had added them in?